shared/middleware/helpers/redis_table.rb

```diff
@@ -110,7 +119,7 @@ def table_from_row_key(key)
   # @param [String] key
   # @return [Integer] the row id.
   def id_from_row_key(key)
-    key.split('_')[1]
+    key.split('_')[1].to_i
```

Comment:
Nice catch; did this actually cause a bug, and does it need to be covered by a test?